starter-code/index.js

```diff
@@ -1,11 +1,75 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor() {
+
+    this._items = [];
+    this.length = 0;
+
+  }
+
+  get items(){
+    return this._items;
+  }
+
+  set items(value){
+    this._items = value;
+    this.length = this._items.length;
+  }
+
+  add(item) {
+
```

Comment:
you seem to have overcomplicated the add function. 

if you remove this code 
``` 
 if(this.length === 0){
      this.items.push(item);
      this.length++;
      return
    } 
```
all the tests still pass. 

Your code is hard to follow here, a cleaner way to write this function would be 

```
add (items) {
    this.items.push(item);
    this.items = this.items.sort((a, b) => a - b);
    this.length = this.items.length;
}

```

Reply:
👍 Right !